pandas/tests/io/formats/test_to_latex.py

```diff
@@ -824,6 +824,33 @@ def test_to_latex_escape_special_chars(self):
         )
         assert result == expected
 
+    def test_to_latex_escape_special_chars_in_index_names(self):
+        special_characters = ["&", "%", "$", "#", "_", "{", "}", "~", "^", "\\"]
```

Comment:
Can you start the first line of the test with a reference to the issue:

    # https://github.com/pandas-dev/pandas/issues/61309